pages/common/routers/success.js

```diff
@@ -1,8 +1,24 @@
+const { merge, get } = require('lodash');
 const { Router } = require('express');
+const successContent = require('../content/success-messages');
 
-module.exports = ({ model = 'model' } = {}) => {
+module.exports = ({
+  model = 'model',
+  licence = 'pel',
+  type = 'amendment'
```

Comment:
I'm not sure having default values here is appropriate. What's the reasoning behind them?

Reply:
most changes are pel amendments

Reply:
but yeah, I'll pass them in